courses/fundamentals_of_ada/020_declarations.rst

```diff
@@ -261,14 +280,12 @@ Decimal Numeric Literals
 Based Numeric Literals
 ------------------------
 
-* Syntax
-
-   .. code:: Ada
+.. code:: Ada
```

Comment:
pygments supports a BNF lexer: https://pygments.org/docs/lexers/#pygments.lexers.grammar_notation.BnfLexer, we should probably declare such code as bnf

Reply:
Currently, everything we put into the RST is understood by Pandoc. If we can get BNF to work "for free" (no installs needed) then I would consider it. However, using "Ada" means all of the keywords are highlighted, which (to me) is a benefit

Reply:
After playing with the BNF syntax color for a bit I agree. First of the BNF is hardly even colored with it, then using the Ada coloration we keep it somewhat consistent between examples, which is a great plus.